### ISSUE TITLE
Exchange tool is not firing BlockBreakEvents

### ISSUE THREAD

**Issue by mcenderdragon** [Direwolf20-MC/BuildingGadgets#43]
I noticed the exchange tool is not firing the block break event. I have protected dungeons so it is quite important for the blocks to stay the same. No bugs have been reported yet as you are firing the Block PlaceEvent and I am blocking that too. But for other who just cover block removal, sending this event [from around here](https://github.com/Direwolf20-MC/BuildingGadgets/blob/master/src/main/java/com/direwolf20/buildinggadgets/items/ExchangerTool.java#L247) would be very nice. Keep up the good work.

**Comment by Direwolf20-MC** [comment]
Hi MCEnderDragon, you'll have to forgive me I'm pretty new to forge, but I thought I was doing a block break?
https://github.com/Direwolf20-MC/BuildingGadgets/blob/master/src/main/java/com/direwolf20/buildinggadgets/items/ExchangerTool.java#L299

**Comment by Direwolf20-MC** [comment]
Thanks @raoulvdberge !!

@mcenderdragon -- Can you test if these 2 commits did the trick for you? If so I'll build and deploy as v1.6.2
https://github.com/Direwolf20-MC/BuildingGadgets/commit/31ac1afc73a45b549f2867c6beb6f224a83d53c1
https://github.com/Direwolf20-MC/BuildingGadgets/commit/6831a0f3c8032cb0ccd429ec46e92a128a4f26e7

**Comment by mcenderdragon** [comment]
Yes this works thanks alot :) And no worries this is absolutly high quallitie mod compatibility, so even alot of large mods dont do this properly.

**Comment by mcenderdragon** [comment]
I honestly don know, what do you mean with "in other languages" , in theorie it can be namend any thing even xyz()
